### PR TITLE
Luxury shuttle now costs 125,000 credits

### PR DESCRIPTION
**Pull request by ToasterBiome**
# Document the changes in your pull request

Joe Captain who's afk in xenobio all round doesn't deserve to dine in such a high class shuttle unless the crew actually makes some money

for a gimmick shuttle it's kinda lame as hell, and picked way too much to make the idea novel instead of annoying

# Changelog

:cl:  
tweak: luxury shuttle increase from 50000 to 150000
/:cl:

### CODE REVIEW

**Comment by N3D6** [comment]
This is better than the one I made

**Comment by Simplehorror** [comment]
This isn't punishing the Captain, just cargo.

**Comment by ToasterBiome** [comment]
> This isn't punishing the Captain, just cargo.

Make some more money then you fat bastard

**Comment by KillerOrcaCora** [comment]
Why is cargo complaining about money?

**Review by adamsong — APPROVED** [review]
Does what it says on the tin

**Comment by LazennG** [comment]
luxury shuttle costs one morbillion credits

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -243,7 +243,7 @@
 	description = "A luxurious golden shuttle complete with an indoor swimming pool. Each crewmember wishing to board must bring 500 credits, payable in cash and mineral coin."
 	extra_desc = "This shuttle costs 500 credits to board."
 	admin_notes = "Due to the limited space for non paying crew, this shuttle may cause a riot."
-	credit_cost = 50000
+	credit_cost = 150000
```
Comment (JamieD1) [inline code/datums/shuttles.dm]:
```suggestion
	credit_cost = 125000
```

**Pull request merged**